drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -4583,13 +4583,24 @@ void RasterizerStorageGLES2::_render_target_allocate(RenderTarget *rt) {
 	GLuint color_internal_format;
 	GLuint color_format;
 	GLuint color_type = GL_UNSIGNED_BYTE;
+	Image::Format image_format;
 
 	if (rt->flags[RasterizerStorage::RENDER_TARGET_TRANSPARENT]) {
+#if defined(ANDROID_ENABLED) || defined(IPHONE_ENABLED)
```

Comment:
What about WebGL? Shouldn't this be `#ifndef GLES_OVER_GL`?

Reply:
Yeah looks like WebGL 1.0 also doesn't support `GL_RGBA8`:
https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/texImage2D

Reply:
sure, but I like #ifdef better so I'll swap em around :)